### DISCUSSION TITLE
Heat source time data

### DISCUSSION THREAD

**Discussion by ali-mhassaan** [idaholab/moose#17616] [Q&A Modules: General] [answered]
Hi,
I'm currently using the FunctionPathEllipsoidHeatSource for a part of my simulation. I need to output the temperature data w.r.t. time and space and use that in another MOOSE input file. I have tried using the NodalValueSampler, but output is not a function of time. Also, any help regarding reading the data (in this particular case), in the input file would also be appreciated.

Cheers,
Hassaan

**Comment by aeslaughter** [comment, marked as answer]
I think you can get what you need from [FunctionAux](https://mooseframework.inl.gov/source/auxkernels/FunctionAux.html). The documentation is not complete, but the test input files listed should give you the enough to get it running.

Using it will result in the calculation of an AuxVariable using the function that will be stored in the Exodus output as a field variable.
